client/app/scripts/liveblog-syndication/directives/consumer-list.js

```diff
@@ -0,0 +1,29 @@
+liveblogSyndication
+    .directive('lbConsumerList', ['api', function(api) {
+        return {
+            templateUrl: 'scripts/liveblog-syndication/views/consumer-list-item.html',
+            scope: {
+                roles: '=',
+                consumers: '=',
+                selected: '=',
+                done: '='
+            },
+            link: function(scope, elem, attrs) {
+                scope.select = function(consumer) {
+                    scope.selected = consumer;
+                };
+
+                scope.disable = function(e, consumerToRemove) {
+                    e.stopPropagation();
+                    //console.log('disable', consumer);
+
+                    api.consumers.remove(consumerToRemove).then(function(result) {
+                        angular.forEach(scope.consumers, function(consumer, i) {
+                            if (consumer._id == consumerToRemove._id)
+                                scope.consumers.splice(i, 1);
```

Comment:
There is no way to break a forEach statement in js https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach